Documentation: improve instructions for scaling configuration

jwkaltz opened the pull request.

sebr72 reviewed (changes requested):
Looks good. Thanks. But it makes sense to merge the 2 commits since the second one is the pre-commit fix.

jwkaltz commented:
@sebr72 I have addressed the requested changes, but GitHub 

> Looks good. Thanks. But it makes sense to merge the 2 commits since the second one is the pre-commit fix.

@sebr72 I have addressed the requested changes, but I don't see how I can resolve the conversation in GitHub. Can you please have a look?
